src/entities/tokenAmount.ts

```diff
@@ -0,0 +1,25 @@
+import { CurrencyAmount, Token } from '@uniswap/sdk-core'
+import invariant from 'tiny-invariant'
+import JSBI from 'jsbi'
+
+import { BigintIsh } from '../../constants'
+
+export class TokenAmount extends CurrencyAmount {

```

Comment:
Do we even need `TokenAmount`? What does it actually add over `CurrencyAmount`?

Reply:
let me remove it

Reply:
I'm just not sure. Does the base class not have addition / subtraction?

Reply:
it does